modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/components/SimpleInputModal.es.js

```diff
@@ -161,9 +175,23 @@ const SimpleInputModal = ({
 								disabled={loadingResponse}
 								id={`${namespace}${mainFieldName}`}
 								name={`${namespace}${mainFieldName}`}
-								onChange={(event) =>
-									setInputValue(event.target.value)
-								}
+								onChange={(event) => {
+									if (errorMessage) {
+										setErrorMessage('');
+									}
```

Comment:
@BarbaraCabrera I this check now necessary? I don't think it is, after L186.

Reply:
It's true, thanks